ERROR - 2024-11-19 16:10:59 --> mysqli_sql_exception: Table 'banco.tipo_transacion' doesn't exist in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\Transacciones.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\Transacciones->index()
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Transacciones))
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#14 {main}
CRITICAL - 2024-11-19 16:10:59 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'banco.tipo_transacion' doesn't exist
[Method: GET, Route: api/Transacciones]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\API\Transacciones.php(14): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\Transacciones->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Transacciones))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 16:10:59 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'banco.tipo_transacion' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tipo_transacion`')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tipo_transacion`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\API\Transacciones.php(14): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\Transacciones->index()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Transacciones))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 16:10:59 --> [Caused by] mysqli_sql_exception: Table 'banco.tipo_transacion' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *
FROM `tipo_transacion`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tipo_transacion`')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tipo_transacion`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\API\Transacciones.php(14): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\Transacciones->index()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Transacciones))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 16:11:52 --> mysqli_sql_exception: Table 'banco.tipo_transacion' doesn't exist in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\TiposTransaccion.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\TiposTransaccion->index()
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\TiposTransaccion))
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#14 {main}
CRITICAL - 2024-11-19 16:11:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'banco.tipo_transacion' doesn't exist
[Method: GET, Route: api/TiposTransaccion]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\API\TiposTransaccion.php(14): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\TiposTransaccion->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\TiposTransaccion))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 16:11:52 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'banco.tipo_transacion' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tipo_transacion`')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tipo_transacion`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\API\TiposTransaccion.php(14): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\TiposTransaccion->index()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\TiposTransaccion))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 16:11:52 --> [Caused by] mysqli_sql_exception: Table 'banco.tipo_transacion' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *
FROM `tipo_transacion`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tipo_transacion`')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tipo_transacion`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\API\TiposTransaccion.php(14): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\TiposTransaccion->index()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\TiposTransaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 16:16:49 --> mysqli_sql_exception: Table 'banco.tipo_transacion' doesn't exist in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\Transacciones.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\Transacciones->index()
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Transacciones))
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#14 {main}
CRITICAL - 2024-11-19 16:16:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'banco.tipo_transacion' doesn't exist
[Method: GET, Route: api/Transacciones]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\API\Transacciones.php(14): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\Transacciones->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Transacciones))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 16:16:49 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'banco.tipo_transacion' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tipo_transacion`')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tipo_transacion`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\API\Transacciones.php(14): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\Transacciones->index()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Transacciones))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 16:16:49 --> [Caused by] mysqli_sql_exception: Table 'banco.tipo_transacion' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *
FROM `tipo_transacion`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tipo_transacion`')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tipo_transacion`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\API\Transacciones.php(14): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\Transacciones->index()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\Transacciones))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 16:30:01 --> mysqli_sql_exception: Table 'banco.tipo_transacion' doesn't exist in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\tipo_transaccion.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->index()
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#14 {main}
CRITICAL - 2024-11-19 16:30:01 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'banco.tipo_transacion' doesn't exist
[Method: GET, Route: api/tipo_transaccion]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\API\tipo_transaccion.php(14): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 16:30:01 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'banco.tipo_transacion' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tipo_transacion`')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tipo_transacion`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\API\tipo_transaccion.php(14): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->index()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 16:30:01 --> [Caused by] mysqli_sql_exception: Table 'banco.tipo_transacion' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *
FROM `tipo_transacion`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tipo_transacion`')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tipo_transacion`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\API\tipo_transaccion.php(14): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->index()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 16:30:51 --> mysqli_sql_exception: Table 'banco.tipo_transacion' doesn't exist in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\tipo_transaccion.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->index()
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#14 {main}
CRITICAL - 2024-11-19 16:30:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'banco.tipo_transacion' doesn't exist
[Method: GET, Route: api/tipo_transaccion]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\API\tipo_transaccion.php(14): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 16:30:51 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'banco.tipo_transacion' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tipo_transacion`')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tipo_transacion`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\API\tipo_transaccion.php(14): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->index()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 16:30:51 --> [Caused by] mysqli_sql_exception: Table 'banco.tipo_transacion' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *
FROM `tipo_transacion`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tipo_transacion`')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tipo_transacion`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\API\tipo_transaccion.php(14): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->index()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 16:30:55 --> mysqli_sql_exception: Table 'banco.tipo_transacion' doesn't exist in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\tipo_transaccion.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->index()
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#14 {main}
CRITICAL - 2024-11-19 16:30:55 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'banco.tipo_transacion' doesn't exist
[Method: GET, Route: api/tipo_transaccion]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\API\tipo_transaccion.php(14): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 16:30:55 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'banco.tipo_transacion' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tipo_transacion`')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tipo_transacion`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\API\tipo_transaccion.php(14): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->index()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 16:30:55 --> [Caused by] mysqli_sql_exception: Table 'banco.tipo_transacion' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *
FROM `tipo_transacion`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tipo_transacion`')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tipo_transacion`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tipo_transacion`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\API\tipo_transaccion.php(14): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->index()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 16:42:57 --> mysqli_sql_exception: Unknown column 'descripicion' in 'field list' in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ti...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ti...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ti...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ti...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\tipo_transaccion.php(22): CodeIgniter\Model->insert(Object(stdClass))
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->create()
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#15 {main}
CRITICAL - 2024-11-19 16:42:57 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'descripicion' in 'field list'
[Method: POST, Route: api/tipo_transaccion/create]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tipo_transaccion` (`descripicion`, `created_at`, `updated_at`) VALUES (:descripicion:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\API\tipo_transaccion.php(22): CodeIgniter\Model->insert(Object(stdClass))
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 16:42:57 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'descripicion' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tipo_transaccion` (`descripicion`, `created_at`, `updated_at`) VALUES (\'debito\', \'2024-11-19 16:42:57\', \'2024-11-19 16:42:57\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tipo_transaccion` (`descripicion`, `created_at`, `updated_at`) VALUES (\'debito\', \'2024-11-19 16:42:57\', \'2024-11-19 16:42:57\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tipo_transaccion` (`descripicion`, `created_at`, `updated_at`) VALUES (:descripicion:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\API\tipo_transaccion.php(22): CodeIgniter\Model->insert(Object(stdClass))
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 16:42:57 --> [Caused by] mysqli_sql_exception: Unknown column 'descripicion' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `tipo_transaccion` (`descripicion`, `created_at`, `updated_at`) VALUES (\'debito\', \'2024-11-19 16:42:57\', \'2024-11-19 16:42:57\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tipo_transaccion` (`descripicion`, `created_at`, `updated_at`) VALUES (\'debito\', \'2024-11-19 16:42:57\', \'2024-11-19 16:42:57\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tipo_transaccion` (`descripicion`, `created_at`, `updated_at`) VALUES (\'debito\', \'2024-11-19 16:42:57\', \'2024-11-19 16:42:57\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tipo_transaccion` (`descripicion`, `created_at`, `updated_at`) VALUES (:descripicion:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\API\tipo_transaccion.php(22): CodeIgniter\Model->insert(Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\tipo_transaccion->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\tipo_transaccion))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 17:29:00 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE) in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transac...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transac...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transac...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transac...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(865): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#15 {main}
CRITICAL - 2024-11-19 17:29:00 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
[Method: PUT, Route: api/transaccion/update/2]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:29:00 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:00\'
WHERE `transaccion`.`id` IN (\'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:00\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:29:00 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:00\'
WHERE `transaccion`.`id` IN (\'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:00\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:00\'
WHERE `transaccion`.`id` IN (\'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 17:29:02 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE) in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transac...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transac...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transac...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transac...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(865): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#15 {main}
CRITICAL - 2024-11-19 17:29:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
[Method: PUT, Route: api/transaccion/update/2]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:29:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:02\'
WHERE `transaccion`.`id` IN (\'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:02\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:29:02 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:02\'
WHERE `transaccion`.`id` IN (\'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:02\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:02\'
WHERE `transaccion`.`id` IN (\'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 17:29:32 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE) in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transac...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transac...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transac...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transac...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(865): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#15 {main}
CRITICAL - 2024-11-19 17:29:32 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
[Method: PUT, Route: api/transaccion/update/2]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:29:32 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:32\'
WHERE `transaccion`.`id` IN (\'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:32\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:29:32 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:32\'
WHERE `transaccion`.`id` IN (\'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:32\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:32\'
WHERE `transaccion`.`id` IN (\'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 17:29:41 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE) in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transac...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transac...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transac...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transac...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(865): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#15 {main}
CRITICAL - 2024-11-19 17:29:41 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
[Method: PUT, Route: api/transaccion/update/2]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:29:41 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:41\'
WHERE `transaccion`.`id` IN (\'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:41\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:29:41 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:41\'
WHERE `transaccion`.`id` IN (\'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:41\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:29:41\'
WHERE `transaccion`.`id` IN (\'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 17:30:30 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE) in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transac...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transac...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transac...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transac...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(865): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#15 {main}
CRITICAL - 2024-11-19 17:30:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
[Method: PUT, Route: api/transaccion/update/2]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:30:30 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:30:30\'
WHERE `transaccion`.`id` IN (\'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:30:30\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:30:30 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:30:30\'
WHERE `transaccion`.`id` IN (\'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:30:30\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:30:30\'
WHERE `transaccion`.`id` IN (\'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 17:30:31 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE) in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transac...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transac...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transac...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transac...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(865): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#15 {main}
CRITICAL - 2024-11-19 17:30:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
[Method: PUT, Route: api/transaccion/update/2]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:30:31 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:30:31\'
WHERE `transaccion`.`id` IN (\'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:30:31\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:30:31 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:30:31\'
WHERE `transaccion`.`id` IN (\'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:30:31\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:30:31\'
WHERE `transaccion`.`id` IN (\'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 17:31:03 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE) in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transac...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transac...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transac...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transac...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(865): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#15 {main}
CRITICAL - 2024-11-19 17:31:03 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
[Method: PUT, Route: api/transaccion/update/2]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:31:03 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:31:03\'
WHERE `transaccion`.`id` IN (\'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:31:03\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:31:03 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:31:03\'
WHERE `transaccion`.`id` IN (\'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:31:03\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'2\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:31:03\'
WHERE `transaccion`.`id` IN (\'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 17:31:57 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE) in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transac...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transac...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transac...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transac...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(865): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#15 {main}
CRITICAL - 2024-11-19 17:31:57 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
[Method: PUT, Route: api/transaccion/update/2]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:31:57 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'3\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:31:57\'
WHERE `transaccion`.`id` IN (\'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'3\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:31:57\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:31:57 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'3\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:31:57\'
WHERE `transaccion`.`id` IN (\'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'3\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:31:57\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'3\', `monto` = \'40.00\', `updated_at` = \'2024-11-19 17:31:57\'
WHERE `transaccion`.`id` IN (\'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 17:41:39 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE) in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `tr...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\transaccion.php(23): CodeIgniter\Model->insert(Object(stdClass))
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#15 {main}
CRITICAL - 2024-11-19 17:41:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
[Method: POST, Route: api/transaccion/create]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (:cuenta_id:, :tipo_transaccion_id:, :monto:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\API\transaccion.php(23): CodeIgniter\Model->insert(Object(stdClass))
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:41:39 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (6, 3, 20, \'2024-11-19 17:41:39\', \'2024-11-19 17:41:39\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (6, 3, 20, \'2024-11-19 17:41:39\', \'2024-11-19 17:41:39\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (:cuenta_id:, :tipo_transaccion_id:, :monto:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\API\transaccion.php(23): CodeIgniter\Model->insert(Object(stdClass))
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:41:39 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (6, 3, 20, \'2024-11-19 17:41:39\', \'2024-11-19 17:41:39\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (6, 3, 20, \'2024-11-19 17:41:39\', \'2024-11-19 17:41:39\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (6, 3, 20, \'2024-11-19 17:41:39\', \'2024-11-19 17:41:39\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (:cuenta_id:, :tipo_transaccion_id:, :monto:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\API\transaccion.php(23): CodeIgniter\Model->insert(Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 17:45:49 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE) in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `tr...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\transaccion.php(23): CodeIgniter\Model->insert(Object(stdClass))
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#15 {main}
CRITICAL - 2024-11-19 17:45:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
[Method: POST, Route: api/transaccion/create]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (:cuenta_id:, :tipo_transaccion_id:, :monto:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\API\transaccion.php(23): CodeIgniter\Model->insert(Object(stdClass))
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:45:49 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (6, 3, 20, \'2024-11-19 17:45:49\', \'2024-11-19 17:45:49\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (6, 3, 20, \'2024-11-19 17:45:49\', \'2024-11-19 17:45:49\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (:cuenta_id:, :tipo_transaccion_id:, :monto:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\API\transaccion.php(23): CodeIgniter\Model->insert(Object(stdClass))
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:45:49 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (6, 3, 20, \'2024-11-19 17:45:49\', \'2024-11-19 17:45:49\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (6, 3, 20, \'2024-11-19 17:45:49\', \'2024-11-19 17:45:49\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (6, 3, 20, \'2024-11-19 17:45:49\', \'2024-11-19 17:45:49\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transaccion` (`cuenta_id`, `tipo_transaccion_id`, `monto`, `created_at`, `updated_at`) VALUES (:cuenta_id:, :tipo_transaccion_id:, :monto:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\API\transaccion.php(23): CodeIgniter\Model->insert(Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:46:52 --> TypeError: App\Models\CustomRules\MyCustomRules::is_valid_cliente(): Argument #1 ($id) must be of type int, string given, called in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Validation\Validation.php on line 338
[Method: POST, Route: api/transaccion/create]
in APPPATH\Models\CustomRules\MyCustomRules.php on line 8.
 1 SYSTEMPATH\Validation\Validation.php(338): App\Models\CustomRules\MyCustomRules->is_valid_cliente('2', null)
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('cliente_id', 'cliente_id', '2', [...], [...], 'cliente_id')
 3 SYSTEMPATH\BaseModel.php(1577): CodeIgniter\Validation\Validation->run([...], null, null)
 4 SYSTEMPATH\BaseModel.php(1000): CodeIgniter\BaseModel->validate([...])
 5 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 6 APPPATH\Controllers\API\transaccion.php(116): CodeIgniter\Model->update(4, [...])
 7 APPPATH\Controllers\API\transaccion.php(25): App\Controllers\API\transaccion->actualizarFondoCuenta(3, 20, 4)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:47:23 --> TypeError: App\Models\CustomRules\MyCustomRules::is_valid_cliente(): Argument #1 ($id) must be of type int, string given, called in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Validation\Validation.php on line 338
[Method: POST, Route: api/transaccion/create]
in APPPATH\Models\CustomRules\MyCustomRules.php on line 8.
 1 SYSTEMPATH\Validation\Validation.php(338): App\Models\CustomRules\MyCustomRules->is_valid_cliente('2', null)
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('cliente_id', 'cliente_id', '2', [...], [...], 'cliente_id')
 3 SYSTEMPATH\BaseModel.php(1577): CodeIgniter\Validation\Validation->run([...], null, null)
 4 SYSTEMPATH\BaseModel.php(1000): CodeIgniter\BaseModel->validate([...])
 5 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 6 APPPATH\Controllers\API\transaccion.php(116): CodeIgniter\Model->update(4, [...])
 7 APPPATH\Controllers\API\transaccion.php(25): App\Controllers\API\transaccion->actualizarFondoCuenta(3, 20, 4)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:51:47 --> TypeError: App\Models\CustomRules\MyCustomRules::is_valid_cliente(): Argument #1 ($id) must be of type int, string given, called in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Validation\Validation.php on line 338
[Method: POST, Route: api/transaccion/create]
in APPPATH\Models\CustomRules\MyCustomRules.php on line 8.
 1 SYSTEMPATH\Validation\Validation.php(338): App\Models\CustomRules\MyCustomRules->is_valid_cliente('2', null)
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('cliente_id', 'cliente_id', '2', [...], [...], 'cliente_id')
 3 SYSTEMPATH\BaseModel.php(1577): CodeIgniter\Validation\Validation->run([...], null, null)
 4 SYSTEMPATH\BaseModel.php(1000): CodeIgniter\BaseModel->validate([...])
 5 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 6 APPPATH\Controllers\API\transaccion.php(116): CodeIgniter\Model->update(4, [...])
 7 APPPATH\Controllers\API\transaccion.php(25): App\Controllers\API\transaccion->actualizarFondoCuenta(3, 20, 4)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:53:21 --> TypeError: App\Models\CustomRules\MyCustomRules::is_valid_cliente(): Argument #1 ($id) must be of type int, string given, called in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Validation\Validation.php on line 338
[Method: POST, Route: api/transaccion/create]
in APPPATH\Models\CustomRules\MyCustomRules.php on line 8.
 1 SYSTEMPATH\Validation\Validation.php(338): App\Models\CustomRules\MyCustomRules->is_valid_cliente('104', null)
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('cliente_id', 'cliente_id', '104', [...], [...], 'cliente_id')
 3 SYSTEMPATH\BaseModel.php(1577): CodeIgniter\Validation\Validation->run([...], null, null)
 4 SYSTEMPATH\BaseModel.php(1000): CodeIgniter\BaseModel->validate([...])
 5 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 6 APPPATH\Controllers\API\transaccion.php(116): CodeIgniter\Model->update(2, [...])
 7 APPPATH\Controllers\API\transaccion.php(25): App\Controllers\API\transaccion->actualizarFondoCuenta(3, 20, 2)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:56:18 --> TypeError: App\Models\CustomRules\MyCustomRules::is_valid_cliente(): Argument #1 ($id) must be of type int, string given, called in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Validation\Validation.php on line 338
[Method: POST, Route: api/transaccion/create]
in APPPATH\Models\CustomRules\MyCustomRules.php on line 8.
 1 SYSTEMPATH\Validation\Validation.php(338): App\Models\CustomRules\MyCustomRules->is_valid_cliente('104', null)
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('cliente_id', 'cliente_id', '104', [...], [...], 'cliente_id')
 3 SYSTEMPATH\BaseModel.php(1577): CodeIgniter\Validation\Validation->run([...], null, null)
 4 SYSTEMPATH\BaseModel.php(1000): CodeIgniter\BaseModel->validate([...])
 5 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 6 APPPATH\Controllers\API\transaccion.php(116): CodeIgniter\Model->update(2, [...])
 7 APPPATH\Controllers\API\transaccion.php(25): App\Controllers\API\transaccion->actualizarFondoCuenta(3, 20, 2)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 17:56:20 --> TypeError: App\Models\CustomRules\MyCustomRules::is_valid_cliente(): Argument #1 ($id) must be of type int, string given, called in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Validation\Validation.php on line 338
[Method: POST, Route: api/transaccion/create]
in APPPATH\Models\CustomRules\MyCustomRules.php on line 8.
 1 SYSTEMPATH\Validation\Validation.php(338): App\Models\CustomRules\MyCustomRules->is_valid_cliente('104', null)
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('cliente_id', 'cliente_id', '104', [...], [...], 'cliente_id')
 3 SYSTEMPATH\BaseModel.php(1577): CodeIgniter\Validation\Validation->run([...], null, null)
 4 SYSTEMPATH\BaseModel.php(1000): CodeIgniter\BaseModel->validate([...])
 5 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 6 APPPATH\Controllers\API\transaccion.php(116): CodeIgniter\Model->update(2, [...])
 7 APPPATH\Controllers\API\transaccion.php(25): App\Controllers\API\transaccion->actualizarFondoCuenta(3, 20, 2)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 18:05:40 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE) in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transac...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transac...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transac...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transac...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(865): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#15 {main}
CRITICAL - 2024-11-19 18:05:40 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
[Method: PUT, Route: api/transaccion/update/2]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 18:05:40 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'3\', `monto` = \'45.00\', `updated_at` = \'2024-11-19 18:05:40\'
WHERE `transaccion`.`id` IN (\'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'3\', `monto` = \'45.00\', `updated_at` = \'2024-11-19 18:05:40\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 18:05:40 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'3\', `monto` = \'45.00\', `updated_at` = \'2024-11-19 18:05:40\'
WHERE `transaccion`.`id` IN (\'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'3\', `monto` = \'45.00\', `updated_at` = \'2024-11-19 18:05:40\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'3\', `monto` = \'45.00\', `updated_at` = \'2024-11-19 18:05:40\'
WHERE `transaccion`.`id` IN (\'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
ERROR - 2024-11-19 18:06:09 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE) in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transac...', 0)
#1 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transac...')
#2 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transac...')
#3 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transac...', Array, false)
#4 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\BaseModel.php(1028): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Model.php(865): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\app\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
#8 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
#9 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
#10 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\rewrite.php(44): require_once('C:\\Users\\136924...')
#15 {main}
CRITICAL - 2024-11-19 18:06:09 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
[Method: PUT, Route: api/transaccion/update/2]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 2 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 18:06:09 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'3\', `monto` = \'45.00\', `updated_at` = \'2024-11-19 18:06:09\'
WHERE `transaccion`.`id` IN (\'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'3\', `monto` = \'45.00\', `updated_at` = \'2024-11-19 18:06:09\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 4 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 18:06:09 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`banco`.`transaccion`, CONSTRAINT `fk_transaccion_cuenta` FOREIGN KEY (`cuenta_id`) REFERENCES `cuenta` (`id`) ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'3\', `monto` = \'45.00\', `updated_at` = \'2024-11-19 18:06:09\'
WHERE `transaccion`.`id` IN (\'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'3\', `monto` = \'45.00\', `updated_at` = \'2024-11-19 18:06:09\'
WHERE `transaccion`.`id` IN (\'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `transaccion` SET `cuenta_id` = \'1\', `tipo_transaccion_id` = \'3\', `monto` = \'45.00\', `updated_at` = \'2024-11-19 18:06:09\'
WHERE `transaccion`.`id` IN (\'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `transaccion` SET `cuenta_id` = :cuenta_id:, `tipo_transaccion_id` = :tipo_transaccion_id:, `monto` = :monto:, `updated_at` = :updated_at:
WHERE `transaccion`.`id` IN :transaccion.id:', [...], false)
 5 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(1028): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\API\transaccion.php(68): CodeIgniter\Model->update('2', Object(stdClass))
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->update('2')
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 19:11:12 --> TypeError: App\Models\CustomRules\MyCustomRules::is_valid_cliente(): Argument #1 ($id) must be of type int, string given, called in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Validation\Validation.php on line 338
[Method: POST, Route: api/transaccion/create]
in APPPATH\Models\CustomRules\MyCustomRules.php on line 8.
 1 SYSTEMPATH\Validation\Validation.php(338): App\Models\CustomRules\MyCustomRules->is_valid_cliente('104', null)
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('cliente_id', 'cliente_id', '104', [...], [...], 'cliente_id')
 3 SYSTEMPATH\BaseModel.php(1577): CodeIgniter\Validation\Validation->run([...], null, null)
 4 SYSTEMPATH\BaseModel.php(1000): CodeIgniter\BaseModel->validate([...])
 5 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 6 APPPATH\Controllers\API\transaccion.php(117): CodeIgniter\Model->update(2, [...])
 7 APPPATH\Controllers\API\transaccion.php(25): App\Controllers\API\transaccion->actualizarFondoCuenta(3, 20, 2)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 19:18:46 --> TypeError: App\Models\CustomRules\MyCustomRules::is_valid_cliente(): Argument #1 ($id) must be of type int, string given, called in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Validation\Validation.php on line 338
[Method: POST, Route: api/transaccion/create]
in APPPATH\Models\CustomRules\MyCustomRules.php on line 8.
 1 SYSTEMPATH\Validation\Validation.php(338): App\Models\CustomRules\MyCustomRules->is_valid_cliente('104', null)
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('cliente_id', 'cliente_id', '104', [...], [...], 'cliente_id')
 3 SYSTEMPATH\BaseModel.php(1577): CodeIgniter\Validation\Validation->run([...], null, null)
 4 SYSTEMPATH\BaseModel.php(1000): CodeIgniter\BaseModel->validate([...])
 5 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 6 APPPATH\Controllers\API\transaccion.php(117): CodeIgniter\Model->update(2, [...])
 7 APPPATH\Controllers\API\transaccion.php(25): App\Controllers\API\transaccion->actualizarFondoCuenta(3, 20, 2)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 19:22:14 --> TypeError: App\Models\CustomRules\MyCustomRules::is_valid_cliente(): Argument #1 ($id) must be of type int, string given, called in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Validation\Validation.php on line 338
[Method: POST, Route: api/transaccion/create]
in APPPATH\Models\CustomRules\MyCustomRules.php on line 8.
 1 SYSTEMPATH\Validation\Validation.php(338): App\Models\CustomRules\MyCustomRules->is_valid_cliente('104', null)
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('cliente_id', 'cliente_id', '104', [...], [...], 'cliente_id')
 3 SYSTEMPATH\BaseModel.php(1577): CodeIgniter\Validation\Validation->run([...], null, null)
 4 SYSTEMPATH\BaseModel.php(1000): CodeIgniter\BaseModel->validate([...])
 5 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 6 APPPATH\Controllers\API\transaccion.php(117): CodeIgniter\Model->update(2, [...])
 7 APPPATH\Controllers\API\transaccion.php(25): App\Controllers\API\transaccion->actualizarFondoCuenta(3, 20, 2)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
CRITICAL - 2024-11-19 19:23:16 --> TypeError: App\Models\CustomRules\MyCustomRules::is_valid_cliente(): Argument #1 ($id) must be of type int, string given, called in C:\Users\13692417\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\system\Validation\Validation.php on line 338
[Method: POST, Route: api/transaccion/create]
in APPPATH\Models\CustomRules\MyCustomRules.php on line 8.
 1 SYSTEMPATH\Validation\Validation.php(338): App\Models\CustomRules\MyCustomRules->is_valid_cliente('104', null)
 2 SYSTEMPATH\Validation\Validation.php(204): CodeIgniter\Validation\Validation->processRules('cliente_id', 'cliente_id', '104', [...], [...], 'cliente_id')
 3 SYSTEMPATH\BaseModel.php(1577): CodeIgniter\Validation\Validation->run([...], null, null)
 4 SYSTEMPATH\BaseModel.php(1000): CodeIgniter\BaseModel->validate([...])
 5 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [...])
 6 APPPATH\Controllers\API\transaccion.php(117): CodeIgniter\Model->update(2, [...])
 7 APPPATH\Controllers\API\transaccion.php(25): App\Controllers\API\transaccion->actualizarFondoCuenta(3, 20, 2)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\API\transaccion->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\API\transaccion))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\Users\\13692417\\Curso-CodeIgniter-Restful-API-Estagio-no-SVGrad\\public\\index.php')
